pkgs/top-level/python-packages.nix

```diff
@@ -13057,6 +13056,7 @@ let
     };
 
     propagatedBuildInputs = with self; [
+      requests
```

Comment:
@domenkozar in [setup.py](https://github.com/michael-lazar/rtv/blob/master/setup.py#L6) is's just `requests` (I'm not familiar with python dependency management).